proto/dex/limit_order_pool_tranche_indexes.proto

```diff
@@ -1,9 +1,9 @@
-syntax = "proto3";
-package nicholasdotsol.duality.dex;
-
-option go_package = "github.com/NicholasDotSol/duality/x/dex/types";
-
-message LimitOrderTrancheTrancheIndexes {
-  uint64 fillTrancheIndex = 1; 
-  uint64 placeTrancheIndex = 2; 
-}
+syntax = "proto3"; 
+package nicholasdotsol.duality.dex; 
+ 
+option go_package = "github.com/NicholasDotSol/duality/x/dex/types"; 
+ 
+message LimitOrderTrancheTrancheIndexes { 
+  uint64 fillTrancheIndex = 1;  
```

Comment:
can rename to `fillIndex` and `placeIndex` since the tranche context is already on the parent.